src/main/java/org/jfrog/bamboo/configuration/AbstractArtifactoryConfiguration.java

```diff
@@ -82,6 +84,7 @@ protected AbstractArtifactoryConfiguration(String builderContextPrefix, @Nullabl
         }
         this.builderContextPrefix = builderContextPrefix;
         this.capabilityPrefix = capabilityPrefix;
+        this.taskConfiguratorHelper = new TaskConfiguratorHelperImpl();
```

Comment:
Can we initialise the instance when it is declared?

Reply:
Done